src/jaxsim/api/ode_data.py

```diff
@@ -116,11 +120,11 @@ class ODEState(JaxsimDataclass):
 
     Attributes:
         physics_model: The state of the physics model.
-        soft_contacts: The state of the soft-contacts model.
+        contacts_state: The state of the contacts model.
     """
 
     physics_model: PhysicsModelState
-    soft_contacts: SoftContactsState
+    contacts_state: js.contact.ContactsState
```

Comment:
Why not just:

```suggestion
    contact: js.contact.ContactsState
```

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/178/commits/4556dbb2b89dbe91b1989903973d93e448e2e8bb

Reply:
Done in https://github.com/ami-iit/jaxsim/pull/178/commits/4556dbb2b89dbe91b1989903973d93e448e2e8bb